src/azure-cli/azure/cli/command_modules/storage/_client_factory.py

```diff
@@ -393,7 +393,7 @@ def cf_share_service(cli_ctx, kwargs):
 
 
 def cf_share_client(cli_ctx, kwargs):
-    return cf_share_service(cli_ctx, kwargs).get_share_client(share=kwargs.pop('share_name'),
+    return cf_share_service(cli_ctx, kwargs).get_share_client(share=kwargs.pop('s_name'),
```

Comment:
Why not change this name? It will affect lots of commands and probably bring in bugs😂